tests/unit/test_transfer_manager.py

```diff
@@ -538,13 +544,13 @@ def test_download_chunks_concurrently():
             worker_type=transfer_manager.THREAD,
         )
     for x in range(MULTIPLE):
-        blob_mock.download_to_file.assert_any_call(
+        blob_mock._prep_and_do_download.assert_any_call(
             mock.ANY,
             **DOWNLOAD_KWARGS,
```

Comment:
Hmmmm I'd imagine `_prep_and_do_download()` here would be called with an overriden value of`"command": "tm.download_sharded"`. Let's take a look at this together:)